### ISSUE TITLE
need to compile build it ?

### ISSUE THREAD

**Issue by LayRiix** [Baseult/Twitchwatcher-Valorant-Drops#3]
there is no exe

**Comment by Fern1ck** [comment]
Yes, you'd need to compile it, although I'm not sure if you have the same issue as I had a couple hours ago. I couldn't compile it because it would throw me a bunch of errors. I couldn't fix it tho

**Comment by LayRiix** [comment]
yeah i tried and got errors

**Comment by SpaceBurn** [comment]
You can get the exe of the discord
